Classes/Core/Acceptance/Helper/AbstractSiteConfiguration.php

```diff
@@ -0,0 +1,66 @@
+<?php
+declare(strict_types=1);
+
+namespace TYPO3\TestingFramework\Core\Acceptance\Helper;
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+use Codeception\Module;
+use TYPO3\CMS\Core\Configuration\SiteConfiguration;
+use TYPO3\CMS\Core\Tests\Acceptance\Support\InstallTester;
+use TYPO3\CMS\Core\Tests\Functional\SiteHandling\SiteBasedTestTrait;
+
+abstract class AbstractSiteConfiguration
+{
+
+    use SiteBasedTestTrait;
+
+    /**
+     * @var array
+     */
+    protected const LANGUAGE_PRESETS = [
+        'EN' => ['id' => 0, 'title' => 'English', 'locale' => 'en_US.UTF8'],
+        'DK' => ['id' => 1, 'title' => 'Dansk', 'locale' => 'da_DK.UTF8'],
+    ];
+
+    /**
+     * @var AcceptanceTester
+     * 
+     * currently unused, but let's keep it for the time being. It will come in handy.
+     */
+    protected $tester;
+
+    public function adjustSiteConfiguration(): void
+    {
+        $sitesDir = __DIR__ . '/../../../../../../../typo3temp/var/tests/acceptance/typo3conf/sites';
```

Comment:
Can we use the ORIGINAL_ROOT constant? seems "flaky" like that.

Reply:
I will try that, but I think you are right.